commands/autoresponse.js

```diff
@@ -0,0 +1,50 @@
+const util = require('../lib/util.js');
+
+const list = require('./autoresponse/list');
+const add = require('./autoresponse/add');
+const remove = require('./autoresponse/remove');
+const info = require('./autoresponse/info');
+
+const command = {};
+
+command.description = 'Adds and lists auto-responses';
+
+command.usage = '<list|add|info|remove> <id>';
+
+command.names = ['autoresponse','response'];
+
+command.execute = async (message, args, database, bot) => {
+    //Permission check
+    if (!message.member.hasPermission('MANAGE_GUILD')) {
+        await message.channel.send('You need the "Manage Server" permission to use this command.');
+        return;
+    }
+    if (!args.length) {
+        return await message.channel.send(await util.usage(message,command.names[0]));
+    }
+
+    let responses = await util.getAutoResponses(message.channel.id, message.guild.id);
```

Comment:
You are loading channel-specific responses here, even though you should load all for the guild.